### DISCUSSION TITLE
1 use alias

### DISCUSSION THREAD

**Discussion by SpencerMT** [Zrips/CMI#7646] [Q&A] [answered]
Just a quick help, new to this stuff, how do I make it so this alias can only be used once, and after that can no longer be used?

![image](https://user-images.githubusercontent.com/27729189/222206884-b14a0f27-eb15-4990-a980-c1c4618ddbda.png)

**Comment by CloudeLecaw** [comment, marked as answer]
you cant create one time aliasesby default.

A little work arround to still have it:
So make perm: true for this alias.
give your players on first join cmi.customalias.raccept and remove the permission as command 2 in your alias.
